source/module_hsolver/kernels/test/perf_math_kernel.cpp

```diff
@@ -0,0 +1,108 @@
+#include "module_base/blas_connector.h"
+#include "module_base/constants.h"
+#include "module_hsolver/kernels/math_kernel_op.h"
+#include "module_psi/kernels/memory_op.h"
+
+#include <complex>
+#include <benchmark/benchmark.h>
+#include <iostream>
+
```

Comment:
Please add annotations for this test:
1. which function is tested?
2. how do you prepare the inputs?
3. how do expand or run this test with different inputs?
4. todo-lists in future for this test?

Reply:
Ok, and I will add them and finish other ops at the same time.